packages/pyright-internal/src/tests/typeEvaluator4.test.ts

```diff
@@ -679,6 +679,12 @@ test('DataClass25', () => {
     TestUtils.validateResults(analysisResults, 17);
 });
 
+test('DataClass26', () => {
```

Comment:
I've been working on cleaning up the test cases, and things have changed a bit here. Please rebase before checking in this change. This new test case should be named `dataclassConverter2.py` to match the new test structure.